src/routes/components/busy-screen/+page.md

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+    import { busyStore } from "$lib/stores/busy.store";
+    import BusyScreen from "$lib/components/BusyScreen.svelte";
+    import { type BusyStateInitiatorType } from "$lib/types/busyState";
+
+    const initiator: BusyStateInitiatorType = "stake-neuron";
+    
+    const startBusy = () => busyStore.startBusy({
+        initiator: initiator,
+        text: 'staking a neuron...',
+    });
+
+    const stopBusy = () => busyStore.stopBusy(initiator);
+
+    const click = (e) => {
+        e.preventDefault();
+        startBusy();
+        setTimeout(() => stopBusy(), 3000);
+    }
+    
+</script>
+
+# BusyScreen
+
+A BusyScreen is a screen that displays a spinning loader and corresponding message (if available), while an application is loading content or performing another process. The component has an absolute centered positioning with the a faded overlay of the screen.
+
+```html
+import {busyStore} from "$lib/stores/busy.store"; import BusyScreen from
```

Comment:
maybe you can follow the toast (https://ipi2f-uqaaa-aaaad-aabza-cai.ic0.app/components/toasts.html) pattern?

in html section the component and then another section usage that display how to use the store?